src/core/utils/i18nutils.js

```diff
@@ -50,3 +50,116 @@ export function parseLocale (localeCode) {
 
   return parsedLocale;
 }
+
+/**
+ * Object containing imperial units
+ */
+const IMPERIAL = {
+  distance: 'mi'
+};
+
+/**
+ * Object containing metric units
+ */
+const METRIC = {
+  distance: 'km'
+};
+
+/**
+ * This object is keyed by language and region. If a region isn't defined, it falls back
+ * to 'default'
+ * @type {Object}
+ */
+export const LOCALE_UNIT_MAP = {
+  en: {
+    GB: METRIC,
+    AU: METRIC,
+    default: IMPERIAL
+  },
+  es: {
+    US: IMPERIAL
+  }
+};
+
+const unitSystemFallback = METRIC;
+
+/**
+ * Convert distance given by liveAPI from meters to kilometers or miles
+ * based on the given locale.
+ *
+ * @param {number} distance
+ * @param {string} locale
+ * @returns {string} localized distance with a unit of length
+ */
+export function localizedDistance (distance, locale) {
+  if (!distance) {
+    return null;
+  }
+  const distanceUnits = _getDistanceUnit(locale);
+  locale = locale.replace(/_/g, '-');
+  switch (distanceUnits) {
+    case 'mi':
+      return _toMiles(distance, locale);
+    case 'km':
+      return _toKilometers(distance, locale);
+    default:
+      return _toMiles(distance, locale);
+  }
+}
+
+/**
+ * Convert distance from meters to kilometers
+ *
+ * @param {number} distance
+ * @param {string} locale
+ * @returns {string} a string consist of distance and km as unit of length
+ */
+function _toKilometers (distance, locale) {
+  const distanceInKilometers = distance / 1000; // Convert meters to kilometers
+  return new Intl.NumberFormat(locale,
+    { style: 'decimal', maximumFractionDigits: 1, minimumFractionDigits: 1 })
```

Comment:
Here and below move `locale` to this line. We either have the args all on one line, or if that's too long, we have one arg per line. I think we should also make sure that `locale` is not flush with `.format`. It should have an additional indent. That makes the code a bit easier to read.